pylint/constants.py

```diff
@@ -173,3 +173,13 @@ class DeletedMessage(NamedTuple):
     # https://github.com/PyCQA/pylint/pull/3571
     DeletedMessage("C0330", "bad-continuation"),
 ]
+
+
+INCOMPATIBLE_WITH_USELESS_SUPPRESSION = [
+    "R0401",  # cyclic-import
+    "W0402",  # deprecated-module
+    "W1505",  # deprecated-method
+    "W1511",  # deprecated-argument
+    "W1512",  # deprecated-class
+    "W1513",  # deprecated-decorator
```

Comment:
I'm wondering how hard it would be to still raise useless-suppression if the function/class/module/decorator... is not deprecated in any python version. I don't think this checker is very performance intensive, maybe relaunching with the highest python version (3.10 right now) is possible ?

Reply:
Though about something similar, but in the context of a bare `disable` once the deprecation has been removed.

> I don't think this checker is very performance intensive, maybe relaunching with the highest python version (3.10 right now) is possible ?

Not sure that is practical. It would almost certainly change major parts of the application flow.
Maybe it's simpler to add a new config option to show **all** `useless-suppression` messages? We should mention however, that it should only be used temporarily and not added to `pylintrc`.

Reply:
@cdce8p I like your last suggestion!